release.cmake

```
@@ -68,7 +69,7 @@ MACRO(RELEASE_SETUP)
       # Update version in package.xml if it exists
       && if [ -f "package.xml" ]; then
           (echo "Updating package.xml to $$VERSION"
-           && sed -i --expression=\"s!<version>[0-9]\\+.[0-9]\\+.[0-9]\\+</version>!<version>$$VERSION</version>!g\" package.xml
+           && sed -i ${SED_I_ARG} -E \"s|<version>.*</version>|<version>$$VERSION</version>|g\" package.xml
```

I think `-E` is useless. If you want to have the same for both mac and linux:
```suggestion
           && sed -i.bak \"s|<version>.*</version>|<version>$$VERSION</version>|g\" package.xml
           && rm package.xml.bak
```

---

You're correct. I apply the change.